modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -423,6 +433,13 @@ public ObjectEntry deleteObjectEntry(ObjectEntry objectEntry)
 			objectEntry.getCompanyId(), objectEntry.getNonzeroGroupId(),
 			objectDefinition.getClassName(), objectEntry.getObjectEntryId());
 
+		if (objectDefinition.isEnableLocalization()) {
```

Comment:
sort it. Put after

```
deleteFromTable(
			objectDefinition.getExtensionDBTableName(),
			objectDefinition.getPKObjectFieldDBColumnName(),
			objectEntry.getObjectEntryId());
```

Reply:
Done [here](https://github.com/liferay-objects/liferay-portal/pull/1913/commits/11e8b5a949679a166c93db5698a1b9643ebcbbb3)